protocol/xmrmaker/swap_state_test.go

```diff
@@ -38,7 +38,7 @@ func newTestSwapStateAndDB(t *testing.T) (*Instance, *swapState, *offers.MockDat
 		types.NewOffer("", new(apd.Decimal), new(apd.Decimal), new(coins.ExchangeRate), types.EthAssetETH),
 		&types.OfferExtra{},
 		xmrmaker.offerManager,
-		coins.NewPiconeroAmount(33),
```

Comment:
33 piconeros won't cover the fees to transfer back in a refund scenario. We should probably have a "minimum-min" XMR offer amount that would prevent funds from getting stranded in a swap wallet, because the amount is too small to transfer to any other wallet.

Reply:
Created an issue for it: https://github.com/AthanorLabs/atomic-swap/issues/286